### DISCUSSION TITLE
Subsequent dimming following turning on.

### DISCUSSION THREAD

**Discussion by Graefer** [Supergiovane/node-red-contrib-knx-ultimate#334] [General]
Hi,
May be rather a feature request than a bug: when I turn on a light and then start dimming, the lamp dims down. In the KNX world I would rather expect the light to dim up in this situation. Am I wrong?
Beside of this: for me no issues anymore! Bravo!
Regards,
Graefe

**Comment by Supergiovane** [comment]
Hi!
I had an user wanting exactly the opposite. Usually, when you turns on a lamp, it goes to 100%, unless customized % has been set. 
Then, when you start dimming, you want the light to dim down. 
I could put an option for that as well, in a future release.

**Comment by Graefer** [comment]
100% when turning on - is this really usually in KNX? Not in my case, I prefer the memory function. 
Anyway, I am well aware, that this is a matter of taste. An option would probably be a very good solution. 
Thanks
